src/server/game.js

```diff
@@ -9,8 +9,6 @@ const permissions = require("./chat/permissions");
 const skins = require("./skins");
 const msgPack = require("msgpack-lite");
 
-const anyColorAllowed = ["abstract", "marble", "default", "swirly", "squares"]; // Will (probably) be changed to a meta file later on, next to the skin file?
-
 function Marble(id, entryId, name, attributes = {}) {
```

Comment:
(Good on ya!)
This should be possible, though looking at it I'll need to rework the way marbles work with the game and physics, since there's a circular dependency related to the `onFinished` callback.
This should ultimately decouple physics and the marbles/game a bit further though, and we'll need that if we ever want to start running the physics on a different thread!